src/validators.ts

```diff
@@ -9,7 +9,7 @@ import {
 import Constants from './constants';
 import { IdentityApiData } from '@mparticle/web-sdk';
 
-type IdentityAPIMethod = 'login' | 'logout' | 'identify' | 'modify';
+export type IdentityAPIMethod = 'login' | 'logout' | 'identify' | 'modify';
```

Comment:
I think it's better to reuse the existing constants rather than redefine new strings. Less of a chance of getting bitten by a typo, IMO. 

Also, not sure if you need to export since I don't think this type is being used outside of this file, but I'm not sure.

```suggestion
type IdentityAPIMethod = valueof<typeof Constants.IdentityMethods>;
```